.github/workflows/build-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Build
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+      - name: Install Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6.1'
+          bundler-cache: true
+      - name: Run Core Tests
+        run: ./gradlew :formula:test
+      - name: Run Kotlin Coroutine Runtime Tests
+        run: ./gradlew :formula-coroutines:test
+      - name: Run Formula Android Unit Tests
+        run: ./gradlew :formula-android:testRelease
+      - name: Run Formula Android Instrumentation Tests
+        run: ./gradlew :formula-android-tests:testRelease
+      - name: Run Formula Lint Tests
+        run: ./gradlew :formula-lint:test
+      - name: Generate Jacoco Report
+        run: ./gradlew jacocoTestReportMerged
+      - name: Save Jacoco Report
+        if: always()
```

Comment:
Is this a legit syntax? I saw `${{ always() }}` in GitHub samples. Also, was the intention here to always run this job, even if some of the previous jobs failed?

Reply:
Reading about it `always()` works as well.